code/datums/datumvars.dm

```diff
@@ -929,6 +929,24 @@
 			manipulate_organs(C)
 			href_list["datumrefresh"] = href_list["editorgans"]
 
+		else if(href_list["hallucinate"])
+			if(!check_rights(0))
+				return
+
+			var/mob/living/carbon/C = locate(href_list["hallucinate"])

```

Comment:
Is there any scope to locate the href? Doing locate in world should be avoided as much as possible.